homeassistant/components/youless/sensor.py

```diff
@@ -151,7 +170,9 @@ def get_sensor(self) -> YoulessSensor | None:
 class PowerMeterSensor(YoulessBaseSensor):
     """The Youless low meter value sensor."""
 
-    _attr_device_class = DEVICE_CLASS_POWER
+    _attr_device_class = DEVICE_CLASS_ENERGY
+    _attr_last_reset = dt.utc_from_timestamp(0)
+    _attr_state_class = STATE_CLASS_TOTAL_INCREASING
```

Comment:
Unit of measurement is missing in order to make this work

Reply:
Just for my information, what is the difference between the attribute for unit of measurement and the attribute for native unit of measurement? The latter was set based upon what the library underneath indicates the UoM to be.

I've updated the code with your suggestions.

Reply:
You'll need to use `_attr_native_unit_of_measurement`

It represents the unit that belongs to the native state this integration provides.